backend/de.metas.adempiere.adempiere/base/src/main/java-legacy/org/compiere/model/PO.java

```diff
@@ -265,6 +265,12 @@ public PO(final Properties ctx, final PO source, final int AD_Client_ID, final i
 		setAD_Org_ID(AD_Org_ID);
 	}    // PO
 
+	/**
+	 * Performance Monitoring sysconfig
+	 */
+	private static final String PERF_MON_SYSCONFIG_NAME = "de.metas.monitoring.po.enable";
+	private static final boolean SYS_CONFIG_DEFAULT_VALUE = false;
```

Comment:
why do we have these constants here (and also the `services.getSysConfigBooleanValue(PERF_MON_SYSCONFIG_NAME, SYS_CONFIG_DEFAULT_VALUE)` checking)

but the actual monitoring methods are in POServicesFacade.

Consider:
* either having all perf related things here
* either having all perf related things in POServicesFacade

but pls don't spread them.